lib/presentation/screens/home/home.dart

```diff
@@ -68,13 +74,13 @@ class _MyHomePageState extends State<MyHomePage> {
       floatingActionButton: FloatingActionButton(
         onPressed: () async {
           await postingsRepo.add(
-            const Post(
-              id: "",
+            PostDraft(
               title: 'title',
               author: 'author',
-              imageUrl:
-                  'https://i1.wp.com/www.suitcasescholar.com/wp-content/uploads/2012/08/DSC_2583.jpg',
-            ),
+              imageFile: File(
```

Comment:
The code here doesn't work.
Probably we want to pass bytes instead of File once we read it from the image picker.
@aliyazdi75 What do you think?

Reply:
One more thing to consider.
Should we separate the image upload and posting in the UI? or do it together?
Should we better create an `upload` method in `postingRepository` or does the `add` method do both?

Reply:
Yes, I think we need to pass the bytes instead.